tests/incremental/01-sideeffects.c

```diff
@@ -1,15 +0,0 @@
-// The patch for this test case is empty on purpose. The problem described does occur in the incremental run even without any changes.
```

Comment:
This comment has disappeared in the renamed copy.

Reply:
I missed that, good that you noticed it. I will reinsert it.